### ISSUE TITLE
Issues in Teaming in LitmusChaos Tutorial

### ISSUE THREAD

**Issue by neelanjan00** [litmuschaos/tutorials#21]
1. Steps for accepting an invitation can also be added on page 5
2. Done button on page 7 redirects to a link which causes a 404 error.

**Comment by mehabhalodiya** [comment]
Hey @neelanjan00 
I would like to contribute.
Can you please specify where to modify in [this](https://github.com/litmuschaos/tutorials/blob/master/tutorials-teaming-in-litmus/index.html)?

**Comment by neelanjan00** [comment]
@mehabhalodiya you can add the steps for accepting a team invite at this place: https://github.com/litmuschaos/tutorials/blob/9e87c961d2f482a47b5e6bd82413a3828cc518f4/tutorials-teaming-in-litmus/index.html#L112

**Comment by mehabhalodiya** [comment]
Oh well, thank you.
One more thing: Is there any reference for adding the steps, which have been mentioned in any other file?
I am a first-time contributor, so just asking :)

**Comment by neelanjan00** [comment]
@mehabhalodiya you can take reference from the docs: https://docs.litmuschaos.io/docs/user-guides/accept-invite
Basically, you can add the same screenshots and descriptions here as well or modify them as per your preference.

**Comment by mehabhalodiya** [comment]
It just came to notice that when I am clicking on `Edit this page` [here]( https://docs.litmuschaos.io/docs/user-guides/accept-invite), it shows me 404 :(

Can you please look on to that?

**Comment by neelanjan00** [comment]
> It just came to notice that when I am clicking on `Edit this page` [here](https://docs.litmuschaos.io/docs/user-guides/accept-invite), it shows me 404 :(
> 
> Can you please look on to that?

@Jonsy13 `v2.1` and `v2.2` docs are not present in the `staging` branch, is this intended or this needs to be fixed?

**Comment by neelanjan00** [comment]
@mehabhalodiya meanwhile if you'd like to edit that page, you can fork the github.com/litmuschaos/litmus-docs repo, edit this file: https://github.com/litmuschaos/litmus-docs/edit/master/website/versioned_docs/version-2.2.0/user-guides/accept-invite.md and then create a PR.

**Comment by Jonsy13** [comment]
@neelanjan00 Yes the branches were not synced. They have been synced now. @mehabhalodiya now you should be able to edit the page in staging. Please follow the instructions by neelanjan for the same.

**Comment by thisisbillall** [comment]
@neelanjan00  can i take Done button 404 issue ?

**Comment by kitarp29** [comment]
Hey @neelanjan00 
I have some Bandwidth I can take over this issue.
Where do we want to redirect users to?

**Comment by neelanjan00** [comment]
> Hey @neelanjan00 
> I have some Bandwidth I can take over this issue.
> Where do we want to redirect users to?

@kitarp29 we can redirect to https://docs.litmuschaos.io or https://docs.litmuschaos.io/tutorials/

**Comment by kitarp29** [comment]
Hey, @neelanjan00 all of our tutorial sites are running into this error. 
I was looking under Google Codelabs as to how to fix it, as these components are loaded automatically from the script.
If there is some resource that you can share it would be of great help.

**Comment by neelanjan00** [comment]
> Hey, @neelanjan00 all of our tutorial sites are running into this error. 
> I was looking under Google Codelabs as to how to fix it, as these components are loaded automatically from the script.
> If there is some resource that you can share it would be of great help.

You're right, currently there's no direct way to configure a redirect path for Codelabs. So the only viable options are to either create an issue in Codelabs and wait for a fix to be implemented or we can redeploy the tutorials to a different URL. CC: @ksatchit @uditgaurav

**Comment by neelanjan00** [comment]
> Hey, @neelanjan00 all of our tutorial sites are running into this error. 
> I was looking under Google Codelabs as to how to fix it, as these components are loaded automatically from the script.
> If there is some resource that you can share it would be of great help.

Feel free to try to mitigate this issue in the Codelabs repo if you like @kitarp29

**Comment by kitarp29** [comment]
https://github.com/googlecodelabs/tools/issues/715
I already did xD

**Comment by neelanjan00** [comment]
> https://github.com/googlecodelabs/tools/issues/715
> I already did xD

Thank you for creating the issue. Please feel free to update here if you come up with any potential solution. Thanks for your contribution.
